crates/rt/src/manifest.rs

```diff
@@ -23,7 +23,7 @@ const DEFAULT_STACK_SIZE: u16 = 2048; // 2 KiB
 /// ```
 #[macro_export]
 macro_rules! manifest {
-    ($($field:ident = $value:tt),*) => {
+    ($($field:ident = $value:expr),* $(,)*) => {
```

Comment:
I would suggest `$(,)?` to match the [standard libraries behaviour](https://sourcegraph.com/github.com/rust-lang/rust@ce85c98575e3016cf2007d90a85be321e592aa96/-/blob/library/core/src/macros/mod.rs?L554) of accepting at most one trailing comma.

Reply:
My bad, I've pushed the commit resolving this issue.